docs/spec/governance/state.md

```diff
@@ -159,7 +159,7 @@ And the pseudocode for the `ProposalProcessingQueue`:
     proposal = load(Governance, <proposalID|'proposal'>) // proposal is a const key
     votingProcedure = load(GlobalParams, 'VotingProcedure')
 
-    if (CurrentBlock == proposal.VotingStartBlock + votingProcedure.VotingPeriod && proposal.CurrentStatus == ProposalStatusActive)
+    if (CurrentTime == proposal.VotingStartTime + votingProcedure.VotingPeriod && proposal.CurrentStatus == ProposalStatusActive)
```

Comment:
Quoting from https://golang.org/pkg/time/#hdr-Monotonic_Clocks:

```
 Note that the Go == operator compares not just the time instant but also the Location and the monotonic clock reading. See the documentation for the Time type for a discussion of equality testing for Time values. 
```

Could this cause problems? I'm not too familiar with the Golang time type. What about using https://golang.org/pkg/time/#Time.Equal instead?

Reply:
Yeah, I assumed this was just pseudocode in the docs.  The actual code uses `{time.Time}.Equal()`